src/domain/email/email.repository.ts

```diff
@@ -39,4 +39,13 @@ export class EmailRepository implements IEmailRepository {
       throw new EmailSaveError(args.chainId, args.safeAddress, args.account);
     }
   }
+
+  async getVerifiedEmailsBySafeAddress(args: {
+    chainId: string;
+    safeAddress: string;
+  }): Promise<string[]> {
```

Comment:
`emailDataSource.getVerifiedSignerEmailsBySafeAddress` returns a `Promise<{ email: string }[]>. I tried to simplify this for the consumers of the repository by mapping it to `string[]`, but no strong opinion regarding this.

Reply:
This is subject to change in the future as the `Email` entities stabilise a bit. Probably this function would return an `Email` instance that would contain more data to be in line with other functions of the Repository.

i.e.: we would work more with an `Email` entity than a `string`.